cmd/client-gen/main.go

```diff
@@ -43,13 +44,13 @@ func init() {
 func ClientGen(cmd *cobra.Command, args []string) error {
 	ctx := context.Background()
 
-	engineConf := &engine.Config{
+	engineConf := engine.Config{
```

Comment:
Made this by-value because there was a place I wanted to assign defaults to the given config, but didn't want to mutate the input.